data.sql

```diff
@@ -73,4 +73,35 @@ WHERE name IN ('Charmander','Squirtle', 'Blossom');
 
 UPDATE animals
 SET owner_id = 5
-WHERE name IN ('Angemon','Boarmon');
\ No newline at end of file
+WHERE name IN ('Angemon','Boarmon');
+
+INSERT INTO vets(name, age, date_of_graduation)
+VALUES
+('William Tatcher',45,'2000-04-23'),
+('Maisy Smith',26,'2019-01-17'),
+('Stephanie Mendez',64,'1981-05-04'),
+('Jack Harkness',38,'2008-06-08');
+
+INSERT INTO specializations(species_id, vets_id)
+VALUES
+(1,1),(1,3),(2,3),(2,4);
+
+INSERT INTO visits(animals_id, vets_id, date_of_visit)
+VALUES
+(1,1,'2020-05-24'),
+(1,3,'2020-07-22'),
+(2,4,'2021-01-02'),
+(3,2,'2020-01-05'),
+(3,2,'2020-03-08'),
+(3,2,'2020-05-14'),
+(4,3,'2021-05-04'),
+(5,4,'2021-02-24'),
+(6,3,'2019-09-29'),
+(7,4,'2020-10-03'),
+(7,4,'2020-11-04'),
+(8,2,'2019-01-24'),
+(8,2,'2019-05-15'),
+(8,2,'2020-02-27'),
+(8,2,'2020-08-03'),
+(9,3,'2020-05-24'),
+(9,1,'2021-01-11');
```

Comment:
- [ ] You missed adding the following data to `visits`, Please make sure to add them.
  - Plantmon visited Maisy Smith on Dec 21st, 2019.
  - Plantmon visited William Tatcher on Aug 10th, 2020.
  - Plantmon visited Maisy Smith on Apr 7th, 2021.

Reply:
Thanks a lot for pointing this out. I will make the correction immediately